tests/full.test.ts

```diff
@@ -702,6 +702,85 @@ for (let index = 0; index < storages.length; index++) {
       expect(rleaf.entry[0].bigInt()).to.be.eq(1n);
     });
 
+    // https://github.com/iden3/go-merkletree-sql/issues/23
+    it('test issue 23', async () => {
```

Comment:
I wouldn't address an issue number in test name. May be it is ok to leave the issue reference in comments but better to name the test by exactly what it does.

Reply:
Renamed